Content.Server/Flash/Components/FlashOnTriggerComponent.cs

```diff
@@ -15,5 +16,10 @@ internal sealed class FlashOnTriggerComponent : Component
         [DataField("duration")] internal float Duration = 8.0f;
 
         internal bool Flashed;
```

Comment:
same with this. repeatable logic should be on the component/system which triggers, not the one which receives the trigger